src/components/SecurityUser/AddSecurityUserComponent.jsx

```diff
@@ -1,78 +1,150 @@
-import React from 'react';
+import React, { Component } from 'react';
 import PropTypes from 'prop-types';
 import { Form, Message } from 'semantic-ui-react';
+
+import emailValidation from '../../_utils/formValidation';
+
 import ArtButton from '../common/ButtonComponent';
+import StatusComponent from '../common/StatusComponent';
+
+class AddSecurityUserComponent extends Component {
+  state = {
+    firstName: '',
+    lastName: '',
+    phoneNumber: '',
+    badgeNumber: '',
+    email: '',
+    emailError: false
+  };
+
+  handleInputChange = (event) => {
+    const { value, name } = event.target;
+    this.setState({
+      [name]: value
+    });
+  };
+
+  handleSubmit = (event) => {
+    event.preventDefault();
+
+    if (!emailValidation(this.state.email)) {
```

Comment:
Hmmm...this `if...else` is bugging me. Let's see...how about using the following:
```jsx
handleSubmit = (event) => {
  event.preventDefault();
  const isValidEmail = emailValidation(this.state.email);
  if (!isValidEmail) {
    return this.setState({ emailError: !isValidEmail });
  }

  this.setState({ emailError: isValidEmail });

  return this.props.addSecurityUser({
    first_name: this.state.firstName,
    last_name: this.state.lastName,
    phone_number: this.state.phoneNumber,
    email: this.state.email,
    badge_number: this.state.badgeNumber
  });
};
```
Regex [test](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/RegExp/test#Returns) method returns a boolean. Therefore, we can use it's value directly

Reply:
Cool. I'll fix that